src/RealtimeChannel.ts

```diff
@@ -23,6 +24,10 @@ export type RealtimeChannelOptions = {
      * key option is used to track presence payload across clients
      */
     presence?: { key?: string }
+    /**
+     * defines if the channel is private or not and if RLS policies will be used to check data
+     */
+    private?: boolean
```

Comment:
@filipecabaco were you able to receive this on the server even though it wasn't explicitly added to the config down here? https://github.com/supabase/realtime-js/blob/828659e9c8bfe9d768489e4aeb4b869b76d87d9a/src/RealtimeChannel.ts#L221